src/saucelabs/job.js

```diff
@@ -33,7 +34,9 @@ export default class Job {
         };
 
         this.requestAdapter = new SaucelabsRequestAdapter(this.options.username, this.options.accessKey);
-        this.platform       = platform;
+        this.browserInfo    = browserInfo;
+        this.platform       = [this.browserInfo.platform || this.browserInfo.platformName || '',
+            this.browserInfo.browserName || '', this.browserInfo.version || this.browserInfo.platformVersion || ''];
```

Comment:
and here you can use `browserInfo.platform` instead of `this.browserInfo.platform` etc..